### PR TITLE
[DD-299]: Update Globe SVG Format and Fix Display in Preview Link 

### PR DESCRIPTION
**Pull request by tshen815**
**Ticket:**[ DD-299](https://circleci.atlassian.net/browse/DD-299)
**Preview Link:** http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/update-globe-svg-preview/?force-all
**Note:** if globe icon is still broken, hard reset and clear browser cache with `cmd + shift + r` 

# Changes
- store `globe.svg` as an asset to be used in img tag
- update style and design of icon 

# Description
We dont currently have a working version of the globe icon for the language selector in our preview links due to not having the correct dynamic img path for our preview builds.

We are also updating the the actual format of the globe icon svg as previously we were including the entire svg directly in our HTML instead of storing it as an asset and referencing it in an img or picture tag. 

# Reasons
See associated PR #6084 

# Screenshots

**Before:**
Screen Shot 2021-12-22 at 5 13 29 PM

**After:**
Screen Shot 2021-12-22 at 5 12 12 PM

### CODE REVIEW

File: src/styles/_global-nav.scss [hunk 1]
```diff
@@ -202,31 +202,34 @@ $btn-selected-blue: #004A95;
...
+      background: none;
+      border: none;
+      padding: 2px 0 0 0;
 
+    .globe-icon {
+      background-color:transparent;
+      color: $black;
+      @media (min-width: $screen-md) {
+        &:hover {
+          filter: invert(39%) sepia(99%) saturate(634%) hue-rotate(105deg) brightness(83%) contrast(97%);
```
Comment (tshen815) [inline src/styles/_global-nav.scss:226]: using filter properties to alter the SVG to get to `#048C43` for button hover.

File: jekyll/_includes/global-nav.html [hunk 2]
```diff
@@ -37,7 +37,7 @@
         {% include_cached mobile-sidebar.html site=site page=page %}
         <div class="global-nav--list global-nav--external-links lang-selector-wrapper">
           <button id="globe-lang-btn" class="global-nav--language-picker" data-analytics-action="click-outer-cta">
-            <span class="globe-icon">{% include utility-icons/globe.svg size="24" %}</span>
+            <img src='/docs/assets/img/compass/globe.svg' class="globe-icon"></img>
```
Comment (ccgeller) [inline jekyll/_includes/global-nav.html:40]: If you put this into preview '/doc's get replaces with preview url and your image won't work. you will want to replace it with {{ site.baseurl }}
```suggestion
            <img src='{{ site.baseurl }}/assets/img/compass/globe.svg' class="globe-icon"></img>
```

File: jekyll/assets/img/compass/globe.svg [hunk 3]
```diff
@@ -1,10 +1,5 @@
-{% if include.size %}
-  {% assign size = include.size %}
-{% else %}
-  {% assign size = 12 %}
-{% endif %}
-
-<svg fill="none" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg">
+<?xml version="1.0" encoding="utf-8"?>
+<svg fill="none" aria-label="Globe" role="img" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg">
```
Comment (ccgeller) [inline jekyll/assets/img/compass/globe.svg:2]: This is just a personal preference feel free to ignore if don't like it but I would put height and width beside each other.
```suggestion
<svg fill="none" aria-label="Globe" role="img" height="18" width="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
```

File: src/js/experiments/forceAll.js [hunk 4]
```diff
@@ -44,5 +44,19 @@ if (
         }
       </style>`,
     );
+
+    const globeBtn = document.getElementById('globe-lang-btn');
+    const globeImg = document.getElementsByClassName('globe-icon');
+    // remove exsisting globe img with broken path from DOM
```
Comment (ccgeller) [inline src/js/experiments/forceAll.js:50]:
```suggestion
    // remove existing globe img with broken path from DOM
```

File: src/js/experiments/forceAll.js [hunk 5]
```diff
@@ -44,5 +44,19 @@ if (
...
+    globeBtn.removeChild(globeImg);
+
+    // create JEKYLL_BASENAME from preview url to use for globe img source
+    const previewURL = jekyllBaseName;
+    const start = previewURL.indexOf('/');
+    const imgPath = previewURL.substring(start);
+    globeBtn.insertAdjacentHTML(
+      'afterbegin',
+      `<img src="/${imgPath}assets/img/compass/globe.svg" class="globe-icon">`,
```
Comment (ccgeller) [inline src/js/experiments/forceAll.js:59]: Should there be anything between ${imgPath} and assets?

File: src/styles/_global-nav.scss [hunk 6]
```diff
@@ -280,6 +277,8 @@ $btn-selected-blue: #004A95;
   }
 }
 
+
+
```
Comment (ccgeller) [inline src/styles/_global-nav.scss:281]:
```suggestion
```

File: jekyll/_includes/global-nav.html (hunk 2 above)
Comment (tshen815) [reply jekyll/_includes/global-nav.html:40]: awesome, thanks for the tip on this! 🎉

**Review by ccgeller — APPROVED** [review]
LGTM! 🤩🥳👑

**Pull request merged**